apps/frontend/src/router.tsx

```diff
@@ -8,6 +8,7 @@ import {
 
 import { PageContainer } from './app/4_templates';
 import { EmailVerificationStateContainer } from './app/4_templates/EmailVerificationStateContainer/EmailVerificationStateContainer';
+import BitocracyPage from './app/5_pages/BitocracyPage/BitocracyPage';
```

Comment:
Use lazy loading
```suggestion
const BitocracyPage = loadable(() => import('./app/5_pages/BitocracyPage/BitocracyPage'));
```
(just make sure it's after imports)